mantidproject/mantid
# Custom output name does not have details within

**Description of work.**
Changes SANS to append the reduction details to a custom output name,
rather than only the detector configuration and phi limit only.

(Sorry about the formatting changes, I only realised after completing all changes)

**Report to:** ISIS/Sarah

**To test:**
- Download the ISIS training data
- Open the ISIS SANS GUI
- Press user file, go to the LOQ demo folder in the training data
- Select maskfile.txt
- Press batch file and select batch_mode_reduction
- Double click `first_one` and delete it (so that it's blank)
- Hit process all
- Look at the workspace names of the processed data
- After the run number of 74044 should be details of the reduction
- After `second_one` should be the same details

Fixes #27316 .

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
